PageSpeed/Test/TestResultsViewController.swift

```diff
@@ -13,10 +13,8 @@ class TestResultsViewController: UIViewController {
     // MARK: - Properties
     var url: String?
     var mobilePageSpeedResult, desktopPageSpeedResult: PageSpeedResponse?
-    var servicesArr: [
-    (id: String, name: String)
-    ]?
-    var gtmetrixResponse: GTMetrixTestStatusResponse?
+    var servicesArr: [(id: String, name: String)]?
+    var gTMetrixResponse: GTMetrixResponse?
```

Comment:
@ilyageekhub once again, `gTMetrixResponse` is incorrect. Should be **gtMetrixResponse**, the same as at https://github.com/ANScorp/pagespeed-swift/pull/5/commits/d5b6264c7201999b5b73ab4b1a0ddc8a0ed097ea#diff-418cd2b81a677cb40ae2f9c2cbd9e28f.
See https://github.com/raywenderlich/swift-style-guide#naming.
>casing acronyms and initialisms uniformly up or down

Reply:
done